lib/employees/schema.ts

```diff
@@ -21,4 +22,17 @@ export default `
   type CreateEmployeeResult {
     ok: Boolean!
   }
+
+  input UpdateEmployeeInput {
+    id: ID!
+    firstName: String!
+    lastName: String!
+    email: String
```

Comment:
```suggestion
    email: String!
```
We want this field to be mandatory, as per the Edit Admin modal design